### PR TITLE
Update to use new FSF files

### PR DESCRIPTION
**Pull request by emma-nechamkin**
#1837

### CODE REVIEW

File: data/data-pipeline/data_pipeline/content/config/csv.yml [hunk 1]
```diff
@@ -310,4 +310,10 @@ fields:
     format: percentage
   - score_name: Does the tract have at least 35 acres in it?
     label: Does the tract have at least 35 acres in it?
+    format: bool
```
Comment (emma-nechamkin) [inline data/data-pipeline/data_pipeline/content/config/csv.yml:313]: This is a hold over mistake!

File: data/data-pipeline/data_pipeline/etl/sources/fsf_flood_risk/etl.py [hunk 2]
```diff
@@ -27,7 +27,7 @@ class FloodRiskETL(ExtractTransformLoad):
     def __init__(self):
         # define the full path for the input CSV file
         self.INPUT_CSV = (
-            self.get_tmp_path() / "fsf_flood" / "flood_tract_2010.csv"
+            self.get_tmp_path() / "fsf_flood" / "flood-tract2010.csv"
```
Comment (emma-nechamkin) [inline data/data-pipeline/data_pipeline/etl/sources/fsf_flood_risk/etl.py:30]: updated with new files

File: data/data-pipeline/data_pipeline/etl/sources/fsf_flood_risk/etl.py [hunk 3]
```diff
@@ -50,24 +50,16 @@ def transform(self) -> None:
...
             dtype={self.INPUT_GEOID_TRACT_FIELD_NAME: str},
             low_memory=False,
         )
 
-        df_fsf_flood_disagg[self.GEOID_TRACT_FIELD_NAME] = df_fsf_flood_disagg[
+        df_fsf_flood[self.GEOID_TRACT_FIELD_NAME] = df_fsf_flood[
             self.INPUT_GEOID_TRACT_FIELD_NAME
         ].str.zfill(11)
 
-        # Because we have some tracts that are listed twice, we aggregate based on
```
Comment (emma-nechamkin) [inline data/data-pipeline/data_pipeline/etl/sources/fsf_flood_risk/etl.py:63]: no longer requires aggregation

File: data/data-pipeline/data_pipeline/etl/sources/fsf_wildfire_risk/etl.py [hunk 4]
```diff
@@ -49,24 +47,16 @@ def transform(self) -> None:
...
             dtype={self.INPUT_GEOID_TRACT_FIELD_NAME: str},
             low_memory=False,
         )
 
-        df_fsf_fire_disagg[self.GEOID_TRACT_FIELD_NAME] = df_fsf_fire_disagg[
+        df_fsf_fire[self.GEOID_TRACT_FIELD_NAME] = df_fsf_fire[
             self.INPUT_GEOID_TRACT_FIELD_NAME
         ].str.zfill(11)
 
-        # Because we have some tracts that are listed twice, we aggregate based on
```
Comment (emma-nechamkin) [inline data/data-pipeline/data_pipeline/etl/sources/fsf_wildfire_risk/etl.py:62]: no longer requires aggregation

**Comment by emma-nechamkin** [comment]
This is going to fail because the fixtures have not been updated. That's fine -- there are a million versions of the fixtures currently and so I'd like to get this approved asap and then iteratively update fixtures since there are so many PRs outstanding.

**Review by esfoobar-usds — APPROVED** [review]
🍏

File: data/data-pipeline/data_pipeline/utils.py [hunk 5]
```diff
@@ -1409,6 +1409,8 @@ def get_excel_column_name(index: int) -> str:
         "ALI",
         "ALJ",
         "ALK",
+        "ALL",
```
Comment (emma-nechamkin) [inline data/data-pipeline/data_pipeline/utils.py:1412]: there's no real reason to add these -- i was testing something -- but no real reason to delete them either :)

**Pull request merged**